raiden-ts/src/raiden.ts

```diff
@@ -89,17 +86,35 @@ export class Raiden {
    */
   public readonly events$: Observable<RaidenEvent>;
 
+  /**
+   * Observable of completed and pending transfers
+   * Every time a transfer state is updated, it's emitted here. 'secrethash' property is unique and
+   * may be used as identifier to know which transfer got updated.
+   */
+  public readonly transfers$: Observable<RaidenSentTransfer>;
+
   /**
    * Expose ether's Provider.resolveName for ENS support
    */
   public readonly resolveName: (name: string) => Promise<Address>;
 
   /**
-   * Observable of completed and pending transfers
-   * Every time a transfer state is updated, it's emitted here. 'secrethash' property is unique and
-   * may be used as identifier to know which transfer got updated.
+   * Get token information: totalSupply, decimals, name and symbol
```

Comment:
This will be generated by typedoc so no need to include in comments

Reply:
Ok, I'll remove and rewrite